Apps/WebClient/src/ClientApp/src/store/modules/medication/modules/statement/actions.ts

```diff
@@ -54,10 +54,12 @@ export const actions: MedicationStatementActions = {
                             );
                             reject(result.resultError);
                         } else {
-                            EventTracker.loadData(
-                                EntryType.Medication,
-                                result.resourcePayload.length
-                            );
+                            if (result.resultStatus === ResultType.Success) {
```

Comment:
it was erroring out when "ActionRequired" protective word was returned

Reply:
good catch!